index.js

```diff
@@ -965,6 +965,27 @@ class Encore {
         return this;
     }
 
+    /**
+     * Configure the style-loader.
+     *
+     * https://github.com/webpack-contrib/style-loader#Options
+     *
+     * ```
+     * Encore.configureStyleLoader(function(config) {
+     *     // change the config
+     *     // config.injectType = 'styleTag';
```

Comment:
This might not be the best example because this is the default value, right?

Reply:
I've done that.